# name of map in common/files/test_maps/
map_name: lab_ipa4.png
# map_name: office_c.png
# map_name: lab_f_scan.png
# map resolution
map_resolution: 0.05
# map origin as X,Y from lower left corner and Z as counter-clockwise rotation (yaw)
map_origin: [19.2, 19.2, 0.0]
# robot_radius
robot_radius: 0.4





